openid4java/1.0.0.wso2v5/pom.xml

```diff
@@ -0,0 +1,192 @@
+                            javax.servlet.*;version="${javax.servlet.imp.pkg.version.range}",
+                            javax.xml.*;version="${javax.xml.version}",
+                            net.sf.ehcache;version="${net.sf.ehcache.imp.pkg.version.range}",
+                            org.apache.commons.codec.*;version="${commons.codec.imp.pkg.version.range}",
+                            org.apache.commons.logging; version="${commons.logging.imp.pkg.version.range}",
+                            org.apache.html.*;resolution:=optional,
+                            org.apache.http.conn.*; version="${http.client.imp.pkg.version.range}",
+                            org.apache.http.auth; version="${http.client.imp.pkg.version.range}",
+                            org.apache.http.client.*; version="${http.client.imp.pkg.version.range}",
+                            org.apache.http.impl.conn.*; version="${http.client.imp.pkg.version.range}",
+                            org.apache.http.impl.client; version="${http.client.imp.pkg.version.range}",
+                            org.apache.http.*; version="${http.imp.pkg.version.range}",
+                            org.apache.xerces.*;resolution:=optional,
+                            org.springframework.*;resolution:=optional,
+                            org.w3c.*;version="${org.w3c.version}",
+                            org.xml.sax.*;version="${org.xml.sax.version}",
+                            com.ibm.icu.*;resolution:=optional,
+                            org.apache.xml.*;resolution:=optional,
+                            org.doomdark.uuid;resolution:=optional,
+                        </Import-Package>
+                        <Embed-Dependency>nekohtml,openxri-syntax,openxri-client;scope=compile|runtime;inline=false
+                        </Embed-Dependency>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
+        <openid4java.version>1.0.0</openid4java.version>
+        <openid4java.export.version>1.0.0</openid4java.export.version>
+        <net.sourceforge.nekohtml.version>1.9.13</net.sourceforge.nekohtml.version>
+        <code.google.imp.pkg.version.range>[1.3.0,1.3.1)</code.google.imp.pkg.version.range>
+        <javax.crypto.version>0.0.0</javax.crypto.version>
+        <javax.net.version>0.0.0</javax.net.version>
+        <javax.servlet.imp.pkg.version.range>[2.6.0,2.7.0)</javax.servlet.imp.pkg.version.range>
+        <javax.xml.version>0.0.0</javax.xml.version>
+        <net.sf.ehcache.imp.pkg.version.range>[1.5,1.6)</net.sf.ehcache.imp.pkg.version.range>
+        <commons.codec.imp.pkg.version.range>[1.4.0,2.0.0)</commons.codec.imp.pkg.version.range>
+        <commons.logging.imp.pkg.version.range>[1.2.0,1.3.0)</commons.logging.imp.pkg.version.range>
+        <http.client.imp.pkg.version.range>[4.5,4.6)</http.client.imp.pkg.version.range>
```

Comment:
any reason not to make the range [4.5, 5.0)?

Reply:
No any specific reason. WIll change it.